ubclaunchpad/nom-appetit
# Style buttons, Load Assets, and Fix Navigation Errors

**Feature/Fix in this PR**
- style basic buttons 
- is now able to load fonts
- fix type errors with navigation

**Code Changes**
(include a short list of all changes made in this PR)
- added new expo-splash-screen module (for loading fonts correctly)
- deleted src/screen/SplashScreen.tsx
- added src/hooks/loadAssets.ts containing function to load fonts
- changed the styling for the basic button in src/styles/Shared.ts
- added src/types/index.ts to fix type issues with the native stack navigator 
- renamed SignUpScreen to SignInScreen (to match Figma screen name)
- added Props interface to all files in src/screen to fix type issue

**Try it Out**
(include a short list of steps to test the feature/fix)
- run npm install to install new dependencies
- run npx react-native-asset to link assets
- the sign in page (first page) should have the correctly styled button with Montserrat font

**Other Notes**
(include any other notes you believe are relevant to this PR)
Main point of this PR was to style buttons. Loading fonts and fixing type issues with navigation was necessary to style the buttons correctly.

**Screenshots**
(include any screenshots you believe are relevant to this PR)
